zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1822,6 +1823,46 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /ad-hoc-activities/search:
+    post:
+      tags:
+        - Ad-hoc subprocess
+      operationId: findAdHocSubprocessActivities
+      summary: Query activatable activities within ad-hoc subprocesses
+      description: |
+        Search for activatable activities within ad-hoc subprocesses based on given criteria.
+
+        Note that this API currently requires filters for both process definition key and ad-hoc
+        subprocess ID and does not support paging or sorting.
```

Comment:
The API tries to align to the other search APIs as much as possible (naming, structure, ...), but request/response models are a subset of the usual search ones without paging + sorting. Will be addressed in #28432.